src/cpp/include/openvino/genai/llm_pipeline.hpp

```diff
@@ -22,6 +23,8 @@ using OptionalGenerationConfig = std::optional<GenerationConfig>;
 using EncodedInputs = std::variant<ov::Tensor, TokenizedInputs>;
 using StringInputs = std::variant<std::string, std::vector<std::string>>;
 
+static constexpr ov::Property<SchedulerConfig> scheduler_config{"scheduler_config"};
```

Comment:
Please, add docstrings to .hpp and python bindings constructors about Continuous batching activation. Openvino docs is the primary source of knowledge. We can influence openvino docs via these docstrings